packages/flutter_tizen/example/tizen/Runner.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen40</TargetFramework>
+    <TargetFramework>tizen80</TargetFramework>
```

Comment:
Changing the TFM may have a side effect. Is there any API you want to use but is not available in API4?

Reply:
As far as I know, there were no APIs that were previously used that are scheduled to be deprecated. And since NUIFlutterView.cs is only used in tizen90(tizen6.5), there were no side effects yet.
When I created a new project and tested it, it worked without any problems. Are there any APIs you're concerned about?

Reply:
I can't say what exact effect there will be but .NET's target framework resolution is quite complicated. The project may not build if the app depends on a library with an incompatible TFM or the embedding is added to an NUI project as a module. It shouldn't be a problem in general, but I just wanted to warn you about the risk.